src/utils/peopleSort.ts

```diff
@@ -0,0 +1,33 @@
+import { Person } from '../types';
+import { SortTypes } from '../types/SortTypes';
+
+export const peopleSort = (
+  people: Person[],
+  sortType: string | null,
+  isReversed: boolean,
+) => {
+  let sortedPeople = [...people];
```

Comment:
You are creating a new array sortedPeople which is unnecessary as sort function doesn't modify the original array but returns a new sorted array.